js/everything.js

```diff
@@ -490,13 +510,16 @@ $(function() {
     // Handle form submission
     $("#form").submit(function(event) {
         event.preventDefault();
-
         var formValue = $.trim($("#v").val());
+        var formValueTime = /&t=(\d+)$/g.exec(formValue);
+        if (formValueTime && formValueTime.length > 1) {
+            formValueTime = parseInt(formValueTime[1], 10);
```

Comment:
Not sure what you mean by needing 2 matches. `formValueTime` returns e.g `["&t=40", "40", index: 11, input: "EBerFisqduk&t=40"]`